src/core/util/config.cc

```diff
@@ -161,7 +161,9 @@ void Configuration::ParseConfig()
   // Help options
   if (m_Map.count("help"))
     {
-      std::cout << config_options << std::endl;
+      // Logging is not setup yet
```

Comment:
Comments should describe what's being done or will be done, not what's *not* being done.

Reply:
I always write comments to why I’m doing things.. what’s being done can be read from the code anyway?

Reply:
>I always write comments to why I’m doing things

Well it's good to get into better habits then. And if you're going to describe why, then describe why. Obviously "logging is not setup yet" but that doesn't explain *why* you need to setup logging.

Reply:
Yea, that comment could have been better.